### PR TITLE
Adjust the access control page for mobile.

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes gitpod-io/gitpod#1855

This is how it looks:

image

### CODE REVIEW

File: components/dashboard/src/components/access-control/access-control.tsx [hunk 1]
```diff
@@ -255,7 +255,7 @@ export class AccessControl extends React.Component<AccessControlProps, AccessCon
                 const oldProviderScopes = oldScopes.get(authProvider.host);
                 const newProviderScopes = newScopes.get(authProvider.host);
                 if (oldProviderScopes && newProviderScopes) {
-                    renderedTokens.push(<Grid item style={{ width: '31%' }}>
+                    renderedTokens.push(<Grid item className="access-control__card-container" style={{ width: '31%'}}>
```
Comment (jankeromnes) [inline components/dashboard/src/components/access-control/access-control.tsx:258]: Nit: Please also move the in-line style (width 31%) into the new CSS class.

**Review by jankeromnes — APPROVED** [review]
Awesome, many thanks for fixing this! Looks and works much better on mobile now. 👍

One tiny nit in-line, otherwise good to merge. 🚀

**Comment by nisarhassan12** [comment]
@jankeromnes I have made the requested changes please have a look again. Whenever you'll have time. Thanks

**Pull request merged**